### PR TITLE
i18n small fixes

### PR DESCRIPTION
**Pull request by evertonfraga**
- It adds translation entries on connectAccount popup
- Also translated to `pt-br` and `es`.

This PR was based on changes at #1414.

### CODE REVIEW

**Comment by luclu** [comment]
Ev, please rebase on develop

**Comment by evertonfraga** [comment]
Yeah, I see that became dirty. But this was based on an ongoing PR (#1414).

**Comment by luclu** [comment]
Travis is having some problem with the german i18n file. It seems like there is one unecessary closing curly bracket (too much). Though I'm having weird merge issues if I try to checkout this PR.

File: interface/i18n/mist.de.i18n.json [hunk 1]
```diff
@@ -186,7 +186,22 @@
                     "unknownFile": "Datei nicht erkannt.",
                     "wrongPassword": "Falsches Passwort.",
                     "importFailed": "Beim Import der Datei ist folgender Fehler aufgetreten: __error__"
-                }
+                },
+            },
+            "daoFork" : {
```
Comment (alexvandesande) [inline interface/i18n/mist.de.i18n.json]: Haven't we removed all of that code already?

**Comment by luclu** [comment]
I created a new *clean* PR here: https://github.com/ethereum/mist/pull/1424

**Pull request closed without merging**